src/pages/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./wishlist.css";
+import { Navbar } from "../../components/Navbar/Navbar";
```

Comment:
u can create an index.js file and then export all the components from that index.js. as all these components are coming from the components folder

Reply:
okay , i will chnage that